js/main.js

```diff
@@ -1120,8 +1124,12 @@ function getLifespan() {
 	const higherDimensions = gameData.taskData["Higher Dimensions"]
 	const abyss = gameData.taskData["Ceaseless Abyss"]
     const cosmicLongevity = gameData.taskData["Cosmic Longevity"]
-    return baseLifespan * immortality.getEffect() * superImmortality.getEffect() * abyss.getEffect()
+    const lifespan = baseLifespan * immortality.getEffect() * superImmortality.getEffect() * abyss.getEffect()
         * cosmicLongevity.getEffect() * higherDimensions.getEffect() * getCompletedGameSpeedBoost()
+
+    if (gameData.active_challenge == "legends_never_die") return Math.pow(lifespan, 0.72) + 365 * 20

```

Comment:
Another way to write this, which would be more in line with player expectations as lifespan is displayed in years:
```suggestion
    if (gameData.active_challenge == "legends_never_die") return Math.pow(lifespan / 365, 0.72) * 365
```

This divides the lifespan by 365 to convert days to years, raises the lifespan (now in years) to the 0.72 power, then multiplies the result by 365 to convert to days since the rest of the game code expects lifespan in days.

Reply:
I don't see the benefit of this tbh. Why bother converting it to days and back when we always store it in days